### DISCUSSION TITLE
Send test reports to a jira issue as a comment.

### DISCUSSION THREAD

**Discussion by fatkobra** [webdriverio/webdriverio#7022] [Q&A]
Hi, 
I am executing my tests and then sending the generated test report to Jira. I have installed a plugin jira-cli that enables me to do that.
The way I am doing it is via package.json:
![image](https://user-images.githubusercontent.com/55045047/122200575-7568fe80-ce9b-11eb-8f23-ac733a9f69f3.png)
`"test": "npx wdio run ./wdio.conf.js && jira comment JIR-571 [https://www.apps.object-storage.com/reports/latest/index.html]"`

Can any of the gurus here teach me a better way of executing this command?
`jira comment JIR-571 [https://www.apps.object-storage.com/reports/latest/index.html]"`

Instead of being as it is now at the package.json, maybe at wdio.conf.js as an after:function?

**Comment by christian-bromann** [comment]
I am not a guru but I can tell you that you can also use [WebdriverIO hooks](https://webdriver.io/docs/options#oncomplete) and a Jira Node.js client to send this data.

**Reply by fatkobra** [reply]
Hey Christian,
I am a tester, not a programer, you mean onComplete hook?
And then something like this?
`onComplete: function(exitCode, config, capabilities, results) {
 jira comment JIR-571 [https://www.apps.object-storage.com/reports/latest/index.html]
},`

Is this the Jira Node.js client: https://www.npmjs.com/package/jira-client
Do you have any sample code?

**Reply by christian-bromann** [reply]
> Do you have any sample code?

I don't , but you might get some inspiration from https://github.com/patthiel/wdio-jira-service

**Reply by fatkobra** [reply]
Excellent, thank you so much.

**Reply by fatkobra** [reply]
@christian-bromann 

This wdio-jira-service is an excellent concept.

I installed it and tried to run, but unfortunately this is outdated.

Does work with: `"@wdio/cli": "^7.6.0",`

**Reply by christian-bromann** [reply]
It would be great to have this updated. It indeed doesn't work with the latest version anymore.
